lib/div-alg.gi

```diff
@@ -1,8 +1,8 @@
 #Ángel
 #####################################################
 # Given a simple algebra output by WedderburnDecompositionInfo
-# or SimpleAlgebraByCharacterInfo from wedderga, 
-# this program determines its actual matrix degree and division 
+# or SimpleAlgebraByCharacterInfo from wedderga,
```

Comment:
I am going to throw away these whitespace changes. Git is unable to make a sense of the diffs and goes into merge conflicts during rebasing.

```
<<<<<<< HEAD
A1:=A;
=======
      if OrderMod(b1,Order(g1))= a1 then
        f:=Order(g1);
#        for m2 in [2..a1] do
         for m2 in [1..f-1] do
          if g1^m2 = E(m1)^c1 then
            c1:=m2;
            break;
          fi;
        od;
        A1:=[m*a2, F, f, [a1,b1 mod f,c1]];
      fi;
>>>>>>> 8c9dd69... Update div-alg.gi

  if Length(A) = 4 then

    F:=A[2];
    a1:=PrimitiveElement(F);
    m1:=A[3];
    a:=A[4][1];
    b:=A[4][2];
    c:=A[4][3];

<<<<<<< HEAD
    n:=Conductor(F);
    if IsOddInt(n) then 
      n:=2*n; 
    fi;
    for m2 in [1..n] do
      if E(n)^m2 in F then
         break;
=======
      if OrderMod(b2,Order(g1))= a2 then
        f:=Order(g1);
#        for m2 in [2..a2] do
          for m2 in [1..f-1] do
          if g1^m2 = E(m1)^c2 then
            c2:=m2;
            break;
          fi;
        od;
        A1:=[m*a1, F, f, [a2,b2 mod f,c2]];
>>>>>>> 8c9dd69... Update div-alg.gi
```

Reply:
ON A SECOND THOUGHT... probably it will take so much time, that I should rather try to create a fresh PR with cumulative changes, let you review it and then merge, rather that trying to refactor and rebase it. Will try now.

Reply:
I agree. I think it is a good moment for all pull the everything and start from scratch. 
Additionally there is a lot of cleaning in the code that can be made: Some commented lines might be removed but there are several comments that are useful to understand the code.
On the other hand indexation may help but that's a boring task!

Reply:
So, let's now close this and continue with #57 - it passed tests, and I compared that the code is the same after applying all changes.

indexation or indentation? That's helpful, but should be done in a separate step. We already suffered enough :)